### PR TITLE
fixed language mismatch for saving translations

### LINKED ISSUE #72: Language mismatches
Translation target languages are currently set to learn language, but should be known language. 
I will go through all of the current `main` code to double check that we're always using the correct language (ie. known or learn) and make the appropriate changes.

### PR DESCRIPTION
**Pull request by mrchrmn**
## Description

Fixes bug where wrong language was save with translation.

## Related Issue

Closes #72 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  :heavy_check_mark:    | :bug: Bug fix              |
|     | :sparkles: New feature     |

### CODE REVIEW

**Comment by mrchrmn** [comment]
Fixed another language mix up when fetching userwords in a text.
Also fixed a bug where phrases starting a sentence would be classified as words instead.

**Comment by eamon0989** [comment]
I've just tried this out on Netlify, when I open a text, no user words are loaded. Is it working correctly for you?

**Comment by eamon0989** [comment]
OK I added some new translations and reloaded and it loaded them perfectly. I'm guessing that means that all my translations in the database were stored under the wrong language then?

**Comment by mrchrmn** [comment]
Yes, it works for my trial user (eamon@example.com).

Maybe you're seeing the result of earlier incorrectly saved translations and words? The database entries are not fixed, of course.

Try a new text maybe?

File: src/components/texts/Body-Paragraph.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ const phrasesState = selector({
 const Sentence = function({ sentence }: { sentence: string }) {
   const phrases = useRecoilValue(phrasesState);
 
+  console.log(phrases);
```
Comment (eamon0989) [inline src/components/texts/Body-Paragraph.tsx:19]: It works, I'll just have to delete the older database translations. Did you forget to remove this console.log?

**Review by eamon0989 — CHANGES_REQUESTED** [review]

File: src/components/texts/Body-Paragraph.tsx (hunk 1 above)
Comment (mrchrmn) [reply src/components/texts/Body-Paragraph.tsx]: I haved removed the console.log.

**Review by eamon0989 — APPROVED** [review]

**Pull request merged**